### PR TITLE
Add method for updating initramfs accordingly to OS version

### PR DESCRIPTION
**Pull request by katlapinka**

### CODE REVIEW

File: test_utils/os_utils.py [hunk 1]
```diff
@@ -8,9 +8,9 @@
 import posixpath
 import re
 import time
-from datetime import timedelta, datetime
 
-from aenum import IntFlag, Enum, IntEnum
+from datetime import timedelta, datetime
+from aenum import IntFlag, Enum, IntEnum, StrEnum
```
Comment (Kamoppl) [inline test_utils/os_utils.py]: ```aenum```
Can`t we just use enum here?

File: test_utils/os_utils.py [hunk 2]
```diff
@@ -102,6 +110,17 @@ class SystemManagerType(Enum):
     systemd = 1
 
 
+def get_distro():
+    output = TestRun.executor.run(
+        "cat /etc/os-release | grep -e \"^NAME=\" | awk -F= '{print$2}' | tr -d '\"'"
+    ).stdout.lower()
+
+    for distro in Distro.__members__.values():
```
Comment (Kamoppl) [inline test_utils/os_utils.py]: I`m not big fan of enum usage like that

File: test_utils/os_utils.py (hunk 1 above)
Comment (Deixx) [reply test_utils/os_utils.py]: indeed current enum version incorporates types that were previously present only in aenum

Comment (katlapinka) [reply test_utils/os_utils.py]: it's outdated, yesterday I changed it to enum :)

File: test_utils/os_utils.py [hunk 3]
```diff
@@ -8,9 +8,10 @@
 import posixpath
 import re
 import time
-from datetime import timedelta, datetime
 
+from datetime import timedelta, datetime
 from aenum import IntFlag, Enum, IntEnum
+from enum import StrEnum
```
Comment (Deixx) [inline test_utils/os_utils.py]:
```suggestion
from enum import IntFlag, Enum, IntEnum, StrEnum
```

File: test_utils/os_utils.py (hunk 1 above)
Comment (Deixx) [reply test_utils/os_utils.py]: I've added what I meant in a new comment :)

**Review by Deixx — APPROVED** [review]

**Review by Kamoppl — APPROVED** [review]

**Pull request merged**